x/auth/mock/simulate_block.go

```diff
@@ -77,25 +77,25 @@ func SignCheckDeliver(t *testing.T, app *baseapp.BaseApp, msg sdk.Msg, seq []int
 	app.EndBlock(abci.RequestEndBlock{})
 
 	// XXX fix code or add explaination as to why using commit breaks a bunch of these tests
-	//app.Commit()
+	app.Commit()
 }
 
 // XXX the only reason we are using Sign Deliver here is because the tests
 // break on check tx the second time you use SignCheckDeliver in a test because
 // the checktx state has not been updated likely because commit is not being
 // called!
-func SignDeliver(t *testing.T, app *baseapp.BaseApp, msg sdk.Msg, seq []int64, expPass bool, priv ...crypto.PrivKeyEd25519) {
-
-	// Sign the tx
-	tx := GenTx(msg, seq, priv...)
-
-	// Simulate a Block
-	app.BeginBlock(abci.RequestBeginBlock{})
-	res := app.Deliver(tx)
-	if expPass {
-		require.Equal(t, sdk.ABCICodeOK, res.Code, res.Log)
-	} else {
-		require.NotEqual(t, sdk.ABCICodeOK, res.Code, res.Log)
-	}
-	app.EndBlock(abci.RequestEndBlock{})
-}
+//func SignDeliver(t *testing.T, app *baseapp.BaseApp, msg sdk.Msg, seq []int64, expPass bool, priv ...crypto.PrivKeyEd25519) {
```

Comment:
If we aren't using this, can we delete it?

Reply:
👍